pkg/transform/transform.go

```diff
@@ -0,0 +1,48 @@
+package transform
+
+import (
+	pb "github.com/expect-digital/translate/api/translate/translate/v1/gen"
+	"github.com/expect-digital/translate/pkg/model"
+)
+
+func MessageFromProto(m *pb.Message) model.Message {
+	return model.Message{
+		ID:      m.Id,
+		Message: m.Message,
+		Fuzzy:   m.Fuzzy,
```

Comment:
Not sure, but shouldn't we use Getters?

Reply:
I make some research and looks like it's not necessary what to use, but since we use proto3-generated files it's better to use fields directly 
https://stackoverflow.com/questions/68235859/should-i-use-get-methods-to-get-values-or-should-i-use-fields-directly